tests/test_extensions/test_ext_intersphinx.py

```diff
@@ -18,6 +18,7 @@
     normalize_intersphinx_mapping,
 )
 from sphinx.ext.intersphinx import setup as intersphinx_setup
+from sphinx.util.console import strip_colors
```

Comment:
Actually, we have twice the same function for that, one in util.console and the other in testing.util `strip_escseq` so we should remember to sync them (I'm not sure whether that's the case).

Reply:
I certainly feel `strip_colors` is the more descriptive name here 👍

Reply:
I just checked and actually `strip_escseq` is lying because it only removes ANSI color codes but not cursor control codes. So the `strip_colors` is good.